index.js

```diff
@@ -325,3 +325,10 @@ Transformer.prototype.renderFileAsync = function (filename, options, locals, cb)
     }.bind(this)), cb);
   }
 };
+
+/**
+ * inputFormats
+ */
+Transformer.prototype.inputFormats = function() {
+  return this._tr.inputFormats || [this._tr.name];
```

Comment:
I think `this._tr.name` should be appended to `this._tr.inputFormats`.

```js
'use strict';

var markdown = require('markdown').markdown;

exports.name = 'markdown-it';
exports.inputFormats = ['md', 'markdown'];
```

Then `transform.inputFormats()` becomes `['md', 'markdown', 'markdown-it']`, allowing to specifically target `markdown-it` easily if you want to target that `inputFormat` directly.